programs/redeem-logic-farming/src/lib.rs

```diff
@@ -135,6 +136,13 @@ impl RedeemLogicConfig {
     16 + // pub interest_split: [u8; 16],
     32 // pub owner: Pubkey,
     ;
+
+    fn dump(&self) {
+        msg!(
```

Comment:
would add a log line more for the section title. something like "redeem logic config:"
I think could help finding the section while scrolling the logs